### PR TITLE
Sdt 502

### PR DESCRIPTION
**Pull request by ian-leggett**
Added the 'Get help with this page pattern'

### CODE REVIEW

File: app/views/index.html [hunk 1]
```diff
@@ -23,6 +23,20 @@ <h1 class="heading-xlarge">HMRC design patterns</h1>
...
+      <h2 class="heading-large">Global patterns</h2>
+
+      <ul class="list">
+        <li><a href="patterns/header/index">Header</a></li>
+        <li><a href="patterns/footer/index">Footer</a></li>
+      </ul>
+
+      <h2 class="heading-large">Patterns</h2>
+      <ul class="list">
+        <li><a href="patterns/get-help-with-this-page/index">Get help with this page</a></li>
```
Comment (rpowis) [inline app/views/index.html]: This is a global pattern. It appears on every page of every service.

File: app/views/patterns/get-help-with-this-page/get-help-with-this-page.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
+{% extends "govuk_template-empty.html" %}
+
+{% block page_title %}
+Header signed in
```
Comment (rpowis) [inline app/views/patterns/get-help-with-this-page/get-help-with-this-page.html]: Is this right? Looks like it's from another page.

File: assets-frontend/stylesheets/application.min.css [hunk 3]
```diff
@@ -3432,7 +3432,7 @@ small {
     }
 }
 
-/*h1,
+h1,
 .h1-heading {
```
Comment (rpowis) [inline assets-frontend/stylesheets/application.min.css]: Looks like these old heading classes aren't being used anywhere. Of not then they can go.

File: app/views/patterns/get-help-with-this-page/get-help-with-this-page.html (hunk 2 above)
Comment (ian-leggett) [inline app/views/patterns/get-help-with-this-page/get-help-with-this-page.html]: This is how Gavin wants to display each pattern when you click on 'Open in new window'

File: assets-frontend/stylesheets/application.min.css (hunk 3 above)
Comment (ian-leggett) [inline assets-frontend/stylesheets/application.min.css]: I think the idea was to leave these in so when we add the pattern to AF we know what needs removing

File: app/views/patterns/get-help-with-this-page/get-help-with-this-page.html (hunk 2 above)
Comment (rpowis) [inline app/views/patterns/get-help-with-this-page/get-help-with-this-page.html]: Ah sorry. My point is `Header signed in` should probably be `Get help with this page` as per the other view

File: assets-frontend/stylesheets/application.min.css (hunk 3 above)
Comment (rpowis) [inline assets-frontend/stylesheets/application.min.css]: But this class is already from AF and is outdated so should be removed (or in this case uncommented?) and the [new heading classes should be used from GOV.UK](https://github.com/alphagov/govuk_elements/blob/master/assets/sass/elements/_elements-typography.scss#L67-L135) instead

**Pull request merged**